Makefile

```diff
@@ -241,6 +253,12 @@ bundle-hub: manifests kustomize operator-sdk ## Generate hub bundle manifests an
 		--patch '[{"op": "add", "path": "/metadata/annotations/olm.skipRange", "value": "$(SKIP_RANGE)"}]' && \
 		$(KUSTOMIZE) edit add patch --name ramen-hub-operator.v0.0.0 --kind ClusterServiceVersion\
 		--patch '[{"op": "replace", "path": "/spec/replaces", "value": "$(REPLACES)"}]'
+	$(SED_CMD) -e "s,channelName: alpha,channelName: $(DEFAULT_CHANNEL)-test," -i config/hub/manifests/$(IMAGE_NAME)/ramen_manager_config_append.yaml
+	$(SED_CMD) -e "s,packageName: ramen-dr-cluster-operator,packageName: $(DRCLUSTER_NAME)," -i config/hub/manifests/$(IMAGE_NAME)/ramen_manager_config_append.yaml
+	$(SED_CMD) -e "s,namespaceName: ramen-system,namespaceName: $(OPERATOR_SUGGESTED_NAMESPACE)," -i config/hub/manifests/$(IMAGE_NAME)/ramen_manager_config_append.yaml
+	$(SED_CMD) -e "s,clusterServiceVersionName: ramen-dr-cluster-operator.v0.0.1,clusterServiceVersionName: $(DRCLUSTER_NAME).v$(VERSION)," -i config/hub/manifests/$(IMAGE_NAME)/ramen_manager_config_append.yaml
```

Comment:
Does ramen-dr-cluster-operator in this line need to be changed based on DRCLUSTER_NAME?

Reply:
Yes, the catalog has the name for the bundle as build using the `$(OSDK) generate bundle -q --package=$(DRCLUSTER_NAME)`

Reply:
I got confused! You are doing it right.